libs/designsystem/menu/src/menu.component.html

```diff
@@ -2,18 +2,23 @@
   <ng-content select="button[kirby-button]"></ng-content>
   <ng-container *ngIf="!userProvidedButton">
     <button
+      #defaultButton
+      id="defaultButton"
       kirby-button
       [size]="buttonSize"
       [disabled]="isDisabled"
       type="button"
       [attentionLevel]="attentionLevel"
-      #defaultButton
+      aria-haspopup="true"
+      [attr.aria-controls]="MENU_ID"
+      aria-expanded="false"
     >
       <kirby-icon [name]="'more'"></kirby-icon>
     </button>
   </ng-container>
 </div>
 <kirby-card
+  [id]="MENU_ID"
```

Comment:
Any specific reason for capital casing? Otherwise consider our regular camel casing used everywhere else:
```suggestion
  [id]="menu-id"
```